app/model/ReminderNotice.scala

```diff
@@ -16,20 +16,35 @@
 
 package model
 
-import model.ProgressStatuses.{ PHASE1_TESTS_FIRST_REMINDER, PHASE1_TESTS_SECOND_REMINDER, ProgressStatus }
+import model.ProgressStatuses._
 
 import scala.concurrent.duration.{ DAYS, HOURS, TimeUnit }
 
 sealed case class ReminderNotice(hoursBeforeReminder: Int, progressStatuses: ProgressStatus) {
```

Comment:
just small observation, don't need to change it:)

with sealed trait you won't need pattern matching and required. Each of the object would deliver exactly the thing the reminder needs